modules/mono/glue/GodotSharp/GodotSharp/Core/AABB.cs

```diff
@@ -54,33 +89,59 @@ public bool Encloses(AABB with)
                    src_max.z > dst_max.z;
         }
 
+        /// <summary>
+        /// Returns this AABB expanded to include a given point.
+        /// </summary>
+        /// <param name="point">The point to include.</param>
+        /// <returns>The expanded AABB.</returns>
         public AABB Expand(Vector3 point)
         {
             Vector3 begin = _position;
             Vector3 end = _position + _size;
 
             if (point.x < begin.x)
+            {
                 begin.x = point.x;
+            }
             if (point.y < begin.y)
+            {
                 begin.y = point.y;
+            }
             if (point.z < begin.z)
+            {
                 begin.z = point.z;
+            }
 
             if (point.x > end.x)
+            {
                 end.x = point.x;
+            }
             if (point.y > end.y)
+            {
                 end.y = point.y;
+            }
             if (point.z > end.z)
+            {
                 end.z = point.z;
+            }
```

Comment:
I don't really like the idea of enforcing braces in C# where the opener goes on a new line. We don't have to follow the C++ code style here. Or do we actually enforce braces in the C# code style and I forgot about it?

This is not a blocker, but I would prefer without it if possible (unless it's from our C# code style and I forgot).

Reply:
The C# style guide doesn't say that braces should always be enforced, but it does say how they should be placed, all examples have braces, and I would totally be in favor of adding this to the style guide. It is more readable in my opinion, and I think being consistent with the C++ code is a good thing.

Reply:
2 lines vs 4 lines for a short `if` is a tough sell with the brace style used for C# though.

I don't think the C# and C++ code styles have to be consistent per se, but I'm not opposed to it either.

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions shows both styles in their examples but predominantly the one with braces though.

Reply:
Ok, I changed some of the parts of the code back to no braces, specifically, the parts in AABB/Rect2 with repeated "return false;" statements, since I think those parts of the code need it the least.

However, I kept other instances of braces, since I think it's more readable and should be the default choice.